### ISSUE TITLE
Labels in combined training to -1 instead of 1

### ISSUE THREAD

**Issue by panquela** [BrainJellyPie/cGANUC#1]
Dear colleague,

I'm wondering if the labels pass to the loss function when training the combined network are 1 instead of -1. I cannot find any in the paper the reason to train it with -1 as your code reflects. 

The following lines is extracted from your code uploaded on last commit
 
 gen_loss = comb.train_on_batch([z, np.concatenate((fs,fs),0)], -1.0*np.ones((bs*2,1)))

The line should be:

gen_loss = comb.train_on_batch([z, np.concatenate((fs,fs),0)], np.ones((bs*2,1)))

Kind regards

Pedro
